### PR TITLE
Adding Slack from Brazil

### PR DESCRIPTION
**Pull request by orafaaraujo**
Adding Slack from Brazil, Android Dev Br, the main community of Android developers who lives in Brazil or speak portuguese.

### CODE REVIEW

**Comment by orafaaraujo** [comment]
Adding Slack from Brazil, Android Dev Br, the main community of Android developers who lives in Brazil or speak portuguese

File: README.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ All those android development sources that you need to be and stay awesome!
 |  | Spec Network - Orthogonal | specnetwork.slack.com |
 | Slack | Local Community |  |
 |  | ADG Berlin | adg-berlin.slack.com |
+|  | Android Dev Br | slack.androiddevbr.org |
```
Comment (ParaskP7) [inline README.md]: Can you please change the URL to a Slack URL instead?

androiddevbr.slack.com

**Review by ParaskP7 — CHANGES_REQUESTED** [review]

**Comment by ParaskP7** [comment]
Thank you so much for the contribution Rafael, much appreciated!

Let's keep this baby up-to-date :)

THANK YOU

**Review by orafaaraujo — COMMENTED** [review]
Updating URL of Android Dev Br

File: README.md (hunk 1 above)
Comment (orafaaraujo) [reply README.md]: Hi, @ParaskP7 
Done!

**Comment by orafaaraujo** [comment]
You're welcome, @ParaskP7 :)

If the community goes well everybody wins!

**Review by ParaskP7 — APPROVED** [review]

**Comment by ParaskP7** [comment]
Exactly!

**Pull request merged**